docs/profile-main.md

```diff
@@ -2,16 +2,16 @@
 
 ## Methods
 
-### profile.load (url, defaultProfile)
+### profile.load (url)
 
   - `url` string - The url of the profile.
-  - `defaultProfile` object - The default profile to use if the profile is not found.
 
 Returns `_Profile` - The Profile instance.
 
 
-Load profile via `url`, if no profile found, it will use the `defaultProfile` and save it to the disk.
+Load profile via `url`.
 You must register your profile path via `profile.register` before you can use it.
+And you can also set the schema - `profile.registerSchema`. This will restrict the data type, If you can't find the data, you'll get the default data from here.
```

Comment:
You also can set a schema for your profile via `profile.registerSchema`. NOTE: only root profile have the schema.